src/components/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import BooksList from '../containers/BooksList';
+import BooksForm from '../containers/BooksForm';
+import store from '../reducers/index';
+
+function App() {
+  return (
+    <Provider store={store}>
```

Comment:
- [ ] It would be better to wrap the children components with the `div` tag
![Screenshot from 2021-10-19 16-28-27](https://user-images.githubusercontent.com/46388113/137952900-895488f4-9989-4c04-a5d3-8106bfe9b25c.png)